samples/multicore/hello_world/power_management_app.overlay

```diff
@@ -1,24 +1,25 @@
 /*
  * Copyright (c) 2024 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Why d we change license here? Isn't it still part of sdk-nrf and we should use Nordic license anyway